exchanges/orderbook/orderbook_types.go

```diff
@@ -79,10 +79,15 @@ type Base struct {
 	LastUpdateID int64         `json:"lastUpdateId"`
 	AssetType    asset.Item    `json:"assetType"`
 	ExchangeName string        `json:"exchangeName"`
+
 	// NotAggregated defines whether an orderbook can contain duplicate prices
 	// in a payload
 	NotAggregated bool `json:"-"`
 	IsFundingRate bool `json:"fundingRate"`
+
+	// ChecksumBypass defines an allowance to bypass internal verification if
+	// the book has been verified by checksum.
+	ChecksumBypass bool `json:"-"`
```

Comment:
I would prefer this to be `HasChecksumValidation`. This name implies that you are bypassing a checksum which is not what this is doing. I was going to suggest `ValidationBypass` but `HasChecksumValidation` is more specific to why you would ever skip validating the orderbook

Reply:
Changed

Reply:
Superfast, thanks!